app/modules/video-players/video-ads.js

```diff
@@ -201,14 +202,30 @@ function init(player, options, slotTargeting) {
 	});
 
 	player.on('adRequest', (event) => {
-		const vastParams = vastParser.parse(event.tag, {
-			imaAd: event.ima && event.ima.ad,
-		});
+		const timeStamp = Math.round(window.performance && window.performance.now && window.performance.timing &&
```

Comment:
Why not `(new Date()).getTime();`?

Reply:
It's faster I think, but we probably don't need those 0.015 microseconds. Changed.